internal/collector/processor/queued/config.go

```diff
@@ -0,0 +1,35 @@
+// Copyright 2019, OpenTelemetry Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package queued
+
+import (
+	"time"
+
+	"github.com/open-telemetry/opentelemetry-service/internal/configmodels"
+)
+
+// ConfigV2 defines configuration for Attributes processor.
+type ConfigV2 struct {
+	configmodels.ProcessorSettings `mapstructure:",squash"`
+
+	// NumWorkers is the number of queue workers that dequeue batches and send them out
```

Comment:
Can you point me to where godoc says that punctuation is important? I cannot find it.

Reply:
My understanding is that structs are shown verbatin in godoc, so this doesn't really matter.

Reply:
Fixed, but I'd like to understand where this requirement is coming from.

Reply:
Effetive Go: "Depending on the context, godoc might not even reformat comments, so make sure they look good straight up: use correct spelling, punctuation, and sentence structure, fold long lines, and so on." https://golang.org/doc/effective_go.html#commentary

Reply:
The closest I can find is this from godoc blog post:
```
Notice this comment is a complete sentence that begins with the name of the element it describes. This important convention allows us to generate documentation in a variety of formats, from plain text to HTML to UNIX man pages, and makes it read better when tools truncate it for brevity, such as when they extract the first line or sentence. 
```
But it doesn't say training dot is important. All it says is that if you have multiple sentences use proper punctuation so that tools can extract the first sentence. That makes sense. However if it is one sentence only it doesn't seem to matter, godoc does not require this.

The only case where it matter is when non-field are published as HTML in godoc and ending the sentence makes them look like complete paragraphs. But this does not apply to fields.

I think it is fine to end them with a dot, but I see that there is plenty of exampes in go standard library where short comments of field have no dot at the end (particularly when they are on the same line).

Reply:
Sounds good. Also Effective Go has this:
"Doc comments work best as complete sentences, which allow a wide variety of automated presentations. The first sentence should be a one-sentence summary that starts with the name being declared. "

Reply:
In any case, I would expect gofmt, govet or golint to enforce this if it is important. This is a job for machines not people. :-)

Reply:
The line length is just crazy in golang in my opinion :)

Reply:
GH duplicating many comments for an already long conversation on a top nit of nitness makes the whole thing even longer...